docs/spelling_wordlist.txt

```diff
@@ -96,6 +96,7 @@ exec
 fastapi
 formatter
 gRPC
+generativeai
```

Comment:
What is this file for?

Reply:
Any words in release notes / docs that are not in the traditional English dictionary should be added here, otherwise sphinx will flag/fail this in the build_docs job: https://sphinxcontrib-spelling.readthedocs.io/en/latest/customize.html#managing-lists-of-correctly-spelled-words-and-ignoring-words